tests/fast/EncryptedBackupCorrectness.toml

```diff
@@ -5,10 +5,6 @@ tenantModes = ['required']
 encryptModes = ['domain_aware']
 
 [[knobs]]
-enable_encryption = true
```

Comment:
there might be a few more workloads that use this knob. I would do a grep and see

Reply:
I did grep. lucky if there's any knob setting remaining, there will be test failure for not able to set the knob.